src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -3477,3 +3477,72 @@ def parain(
             scale=scale,
             **kwargs,
         )
+
+    def screenshot(self, savefig: Optional[str] = None):
+        """Screenshot Take an MAPDL screenshot and show it in a popup window.
+
+        Parameters
+        ----------
+        savefig : Optional[str], optional
+            Name of or path to the screenshot file.
+            The default is ``None``.
+
+        Returns
+        -------
+        str
+            File name.
+
+        Raises
+        ------
+        FileNotFoundError
+            If the path given in `savefig` is not found or is not consistent.
+        ValueError
+            If given a wrong type for `savefig` parameter.
```

Comment:
```suggestion
            If given a wrong type for the ``savefig`` parameter.
```
Single backtick just displays as italics in the doc. Double backticks display as a code entity.